extensions/wikia/EditPageLayout/js/plugins/Leaveconfirm.js

```diff
@@ -37,16 +37,16 @@
 		},
 
 		onEditorReady: function() {
-			this.initialContent = this.editor.getContent();
+					this.initialContent = this.editor.getContent();
 		},
 
 		onMarkDirty: function() {
-			this.isDirtyFlag = true;
+					this.isDirtyFlag = true;
 			this.editor.log('edit page marked as dirty');
 		},
 
 		onStateChange: function(editor, state) {
-			var states = editor.states;
+					var states = editor.states;
 
 			switch (state) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)

[project-id]: wikia:app
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement with "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1301)

[project-id]: wikia:app